README.md

```diff
@@ -10,30 +10,93 @@ Neutrino + Elements
 
 ## Overview
 
-neutrino-elements uses Compact Block Filter (BIP0158) to implement a light client for [elements](https://elementsproject.org/)-based networks.
+Neutrino-elements is a set of useful packages and binaries that can be used to "watch" Liquid side-chain events.<br>
+It uses Compact Block Filter (BIP0158) to implement a light client for [elements](https://elementsproject.org/) based networks.<br>
 
-Two services are provided, they can work independantly:
-- `NodeService` is a full node maintaining an up-to-date state of the block headers + compact filters. The NodeService writes down headers and filters in repositories.
-- `ScannerService` uses filters and headers repositories to handle `ScanRequest` which aims to know if an outpoint (identified by its script) is spent or not. 
+Two packages, that works independently, are provided if you want to build your own light client:<br>
+- `NodeService` is a full node maintaining an up-to-date state of the block headers + compact filters. The NodeService writes down headers and filters in repositories.<br>
+- `ScannerService` uses filters and headers repositories to handle `ScanRequest` which aims to know if an outpoint (identified by its script) is spent or not.<br>
 
-## Getting Started
+Two binaries are provided if you want to use ready light client:<br>
+- `neutrinod` is a daemon that accepts websocket connections on which clients can send requests to watch for events related to wallet-descriptor<br>
+neutrinod also exposes HTTP endpoint that supports adding webhooks to receive notifications when an event occurs.<br>
+- `neutrino` is a simple command line tool that can be used to watch Liquid side-chain events.<br>
 
-### Build
+## Local Development
+
+### Requirements
+- Go (^1.17.*)
+- Nigiri (^0.4.*)
+
+### Build neutrinod & neutrino CLI
 
 ```
-make build
+make build-nd
+make build-n
 ```
 
-### Unit tests
+### Run test
 
 ```
 make test
 ```
 
-### Format
+## Usage
+
+### Run neutrinod
+
+```
+./bin/neutrinod
+```
+
+### Config CLI
+```
+./bin/neutrino config
+```
+
+### Watch for events related to wallet-descriptor(websocket-{HOST:PORT}/neutrino/subscribe/ws) 
+```
+./bin/neutrino subscribe --descriptor="{WALLET_DESCRIPTOR}" --block_height={BLOCK_HEIGHT}
+```
+### Add webhook(http)
+```
+curl -X POST http://localhost:8080/neutrino/subscribe/http \
+   -H 'Content-Type: application/json' \
+   -d '{"actionType":"register", "eventTypes":[0], "descriptorWallet":"wpkh(037470e26cc774eca62ca19e1a182461a5f3d3680acbc593ce3f38cd142c26c03d)", "startBlockHeight":0, "endpointUrl":"http://127.0.0.1:62900"}'
```

Comment:
eventTypes passed as number are not documented anywhere. I think is best to have human readable string here in the user-facing API